Making changes suggested by our docs writers to MongoDB and redis sed (#11)
redhat-developer/service-endpoint-definition: seds/mongodb-sed/README.md

> This helm chart defines a MongoDB Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:

Reviewer: ```suggestion
This Helm chart defines a MongoDB Service Endpoint Definition (SED). When the SED chart is installed, it provides the user with the opportunity to provide binding data and credentials to authenticate. The following values must be customized while installing the SED chart:
```

Suggestion: The H in Helm is always in capitals. Global comment. Please change in all such instances.

1. Hostname
1. Port
1. Username
1. Password
1. Databasename

> The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MongoDB SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).

Reviewer: ```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MongoDB SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Suggestion: Apply the comment to Redis SED content also.
